packages/@unimodules/core/android/src/main/java/org/unimodules/core/interfaces/SingletonModule.java

```diff
@@ -1,5 +1,5 @@
 package org.unimodules.core.interfaces;
 
-public interface SingletonModule {
+public interface SingletonModule extends ComponentLifecycleListener{
```

Comment:
```suggestion
public interface SingletonModule extends ComponentLifecycleListener {
```

Please configure autoformatter for your Android Studio